app/views/users/form.html.erb

```diff
@@ -20,6 +20,17 @@
     <%= f.field(admin_mode? ? :assignments : :role, :partial => "assignments", :required => !admin_mode?) %>
   <% end %>
 
+  <% if @user.user_groups.present? %>
+    <div class="form_field user_groups">
+      <div class="label_and_control">
+        <label class="main"><%= I18n.t('activerecord.attributes.user.user_groups') %>:</label>
+        <div class="control read_only">
+          <%= @user.user_groups.map(&:name).join(', ') %>
+        </div>
+      </div>
+    </div>
```

Comment:
I think there should be a way to do this using `f.field`. How about `f.field(:group_names, read_only: true)`, and you add a `group_names` method to user?